lca-cli/seedcreator/seedcreator.go

```diff
@@ -96,6 +97,7 @@ func (s *SeedCreator) CreateSeedImage() error {
 	if err := utils.RunOnce("gather_cluster_info", common.BackupChecksDir, s.log, s.gatherClusterInfo, ctx); err != nil {
 		return fmt.Errorf("failed to run once gather_cluster_info: %w", err)
 	}
+	s.ensureRecertImageWithDigestFormat()
```

Comment:
Too late, `gatherClusterInfo` already [establishes](https://github.com/openshift-kni/lifecycle-agent/blob/99c6a4cb46939629e793dbaa832cc2bd2d1c562d/lca-cli/seedcreator/seedcreator.go#L189) the seed cluster info (including which recert image seed uses) one line above:

https://github.com/openshift-kni/lifecycle-agent/blob/99c6a4cb46939629e793dbaa832cc2bd2d1c562d/lca-cli/seedcreator/seedcreator.go#L97-L99

Reply:
Thanks, fixedd